src/app/list-item/list-item.component.html

```diff
@@ -0,0 +1,14 @@
+<div *ngFor="let item of items">
+    <div class="item">
+        <div class="item-header">
+            <h1 class="item-title">{{ item.title }}</h1>
+            <div class="item-list">
+                <img class="icon-registry" src="https://img.icons8.com/pastel-glyph/64/000000/spiral-bound-booklet.png">
+                <p class="list">{{ item.list }}</p>
+            </div>
+        </div>
+        <div class="item-content">
+            <p class="item-descr">{{ item.description }}</p>
+        </div>
+    </div>
+</div>
```

Comment:
All this code can be simplifyed reusing the already defined item comonent (not copy & pasting its content).
`<app-todo-item *ngFor="let item of items" [item]="item"></app-todo-item>`

The only change you should do to app-todo-itm is managint its item as an angular `@Input`